integration/generate_pipeline_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
+)
+
+func TestGeneratePipeline(t *testing.T) {
+	tests := []struct {
+		description string
+		dir         string
+		args        []string
+		responses   []byte
+	}{
+		{
+			description: "no profiles",
+			dir:         "testdata/generate_pipeline/no_profiles",
+			responses:   []byte("y"),
+		},
+		{
+			description: "existing oncluster profile",
+			dir:         "testdata/generate_pipeline/existing_oncluster",
+			responses:   []byte(""),
+		},
+		{
+			description: "existing other profile",
+			dir:         "testdata/generate_pipeline/existing_other",
+			responses:   []byte("y"),
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
```

Comment:
Please add 			if testing.Short() {
				t.Skip("skipping integration test")
			}
			if ShouldRunGCPOnlyTests() {
				t.Skip("skipping test that is not gcp only")
			}